src/main/java/ru/ayaz/invalidNicknameException.java

```diff
@@ -0,0 +1,15 @@
+package ru.ayaz;
+
+public class invalidNicknameException extends Exception {
```

Comment:
&#x1F34E; **Issue**
Classes should always be named starting with a capital letter.